src/widgets/scroll.rs

```diff
@@ -98,6 +98,30 @@ impl Scroll {
         self
     }
 
+    /// Creates a reader for the scroll value.
+    pub fn get_scroll(&self) -> DynamicReader<Point<Px>> {
```

Comment:
All the `get_` functions should just be called whatever they are getting ([see Rust API Guidelines](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter)). For this specifically, though, I would skip having an accessor.